web-rdfs/source/http_handlers.cc

```diff
@@ -77,25 +95,25 @@ void get_handler(std::shared_ptr<HttpsServer::Response> response,
     delete_file_handler(response, path);
   } else if (!typeOfRequest.compare("OPEN")) {
     read_file_handler(response, path);
-  } else {
-    create_file_handler(response, request);
+  } else if (!typeOfRequest.compare("CREATE")){
+    create_file_handler(response, path);
+  }
+  else {
+    ls_handler(response, request);
```

Comment:
Might be better to make this an else if just so you make sure ls only occurs on LISTSTATUS